src/compiler/Restler.Compiler/Utilities.fs

```diff
@@ -57,7 +57,8 @@ module JsonSerialization =
                 JsonSerializerSettings(
                     NullValueHandling = NullValueHandling.Ignore,
                     MissingMemberHandling = MissingMemberHandling.Error,
-                    DateParseHandling = DateParseHandling.None
+                    DateParseHandling = DateParseHandling.None,
+                    ContractResolver = Compact.Strict.RequireNonOptionalPropertiesContractResolver()
```

Comment:
>RequireNonOptionalPropertiesContractResolver [](http://example.com/codeflow?start=54&length=44)

This type from the FSharpLU project inherits from CamelCasePropertyContractResolver, which is not desired here.  You could define your own type in this 'JsonSerialization' module that is identical except inherits from the default contract resolver:

```
        /// A contract resolver that requires presence of all properties
        /// that are not of type option<_>
        type RequireNonOptionalPropertiesContractResolver() =
            inherit Newtonsoft.Json.Serialization.DefaultContractResolver()
            override __.CreateProperty(_member, memberSerialization) =
                let property = base.CreateProperty(_member, memberSerialization)
                let isRequired = not (property.PropertyType.GetTypeInfo().IsGenericType 
                                   && property.PropertyType.GetGenericTypeDefinition() = typedefof<option<_>>)
                if isRequired then 
                    property.Required <- Required.Always
                    property.NullValueHandling <- System.Nullable NullValueHandling.Ignore
                property

```

Reply:
It looks like the base type [now inherits from `DefaultContractResolver`](https://github.com/microsoft/fsharplu/blob/805dce44eb8b4110a1a03ce835a3fe74dcc0eaff/FSharpLu.Json/Compact.fs#L439) (and has since [2019](https://github.com/microsoft/fsharplu/commit/5ff9741f073d81ba51ed445bfa66d009b457142a), where camelCasing was moved from `NonOptional...` to `CompactStringSettings`), so this local definition exactly matches the one upstream.

To confirm, I tried a local definition which inherits from `CamelCasePropertyContractResolver` - that fails several unit tests. So the version in this PR, which passes those unit tests, must be using `DefaultContractResolver` already.

Happy to move this definition inline if there's a reason not to rely on the upstream, but it looks unnecessary (and protected by unit tests) at the moment.

Reply:
Ah, my bad!  I must have looked at a very old version on GitHub, not sure how that happened.  Yes, looks good, no reason to move the definition inline.